docs/CHANGELOG.md

```diff
@@ -17,7 +17,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Experimental support of runtime process isolation for Solana data feeds. Requires plugin binaries to be installed and
   configured via the env vars `CL_SOLANA_CMD` and `CL_MEDIAN_CMD`. See [plugins/README.md](../plugins/README.md).
-
+- Add a new field called`Priority` to `EVM.Nodes` that is used for the `PriorityLevel` node selector and also as a tie-breaker for `HighestHead` and `TotalDifficulty`. Lower number means higher priority. If priority is not defined it will default to `0` (highest priority).
+- Added new node selection mode called `PriorityLevel` for EVM, it is a tiered round-robin where the highest priority is favored. Example:
```

Comment:
Maybe avoid the ambiguous words 'lowest' and `highest`? :shrug: 
```suggestion
- Add a new field called`Priority` to `EVM.Nodes` that is used for the `PriorityLevel` node selector and also as a tie-breaker for `HighestHead` and `TotalDifficulty`. `Priority` levels are considered in ascending order. If not defined it will default to `Priority = 0` (first level).
- Added new node selection mode called `PriorityLevel` for EVM, it is a tiered round-robin in ascending order of `Priority`. Example:
```

Reply:
Updated in [`b6c1131` (#9329)](https://github.com/smartcontractkit/chainlink/pull/9329/commits/b6c113165da19531378dc8e005a5bf85524253a6)